bigtable/spark/src/main/scala/example/Wordcount.scala

```diff
@@ -0,0 +1,61 @@
+package example
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration
+import org.apache.hadoop.hbase.client._
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable
+import org.apache.hadoop.hbase.mapreduce.TableOutputFormat
+import org.apache.hadoop.hbase.util.Bytes
+import org.apache.spark.SparkContext
+
+// FIXME Explain the purpose of the app
+object Wordcount extends App {
+
+  val projectId = args(0)
+  val instanceId = args(1)
+  val table = args(2)
+  val file = args(3)
+
+  val ColumnFamily = "cf"
+  val ColumnFamilyBytes = Bytes.toBytes(ColumnFamily)
+  val ColumnNameBytes = Bytes.toBytes("Count")
+
+  var hConf = BigtableConfiguration.configure(projectId, instanceId)
+  hConf.set(TableOutputFormat.OUTPUT_TABLE, table)
+
+  import org.apache.hadoop.mapreduce.Job
+  val job = Job.getInstance(hConf)
+  job.setOutputFormatClass(classOf[TableOutputFormat[ImmutableBytesWritable]])
+  hConf = job.getConfiguration
+
+  import org.apache.spark.SparkConf
+  val config = new SparkConf()
+
+  // Workaround for a bug in TableOutputFormat
+  // See https://stackoverflow.com/a/51959451/1305344
+  // Without the property:
```

Comment:
because this will be used as a sample, I'd prefer to keep this a little less verbose. I think it's good to mention it's a workaround, but I would leave out the stack trace.